docs/dev/feedback-json.rst

```diff
@@ -0,0 +1,76 @@
+.. _feedback-json-doc:
+
+Feedback JSON Documentation
+===========================
+
+.. note:: Do not submit copyrighted information in these fields!
+
+Submission JSON
+---------------
+
+To submit a feedback via our API (see: :doc:`api`), ``POST`` a JSON document to
+the ``submit-feedback`` endpoint.
+
+   - The complete JSON document may not exceed :data:`~webserver.views.api.MAX_FEEDBACK_SIZE` 
```

Comment:
This is also not needed since a feedback object is nearly fixed in size.

Reply:
Removed